util/converter.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Convert class names to style object
+ *
+ * 'px-10 -mt-20 h-1/2'
+ * is equivalent to [t.px10, t._mt20, h1_2]
+ *
+ * Returns an object instead of array
+ * Later styles can override previous and that's what you expect
+ * 'px-10 px-16' is equal to 'px-16'
+ */
+import t from '../tailwind';
+
+const cnToStyleWithCache = () => {
+  // save cache object to make the function little bit faster
+  const styleCache = {};
+
+  return (classes = '') => {
+    let style = {};
+    if (styleCache[classes]) return styleCache[classes];
+
+    classes.split(' ').map((className) => {
+      if (styleCache[className]) return style = { ...style, ...styleCache[className] };
+
+      let cn = className;
+      const isNegative = cn.startsWith('-');
```

Comment:
can't we use translateKeys from utils/generator directly ?
 https://github.com/TVke/react-native-tailwindcss/blob/32fc989ff22cd8c8bc28888765f94d16c324bac9/util/generator.js#L249

Reply:
Ohh great. I didn't found this. 😅 I will do this for sure.